pallets/dao/src/tests.rs

```diff
@@ -519,24 +520,24 @@ fn user_can_be_removed_from_organization_it_belongs_to_member_of() {
 		const ORG_NAME2: &'static [u8] = &[8];
 
 		// Ensure organization can be created
-		assert_ok!(Dao::create_organization(Origin::signed(1), ORG_NAME1.to_vec()));
-		assert_ok!(Dao::create_organization(Origin::signed(1), ORG_NAME2.to_vec()));
+		assert_ok!(Dao::create_organization(Origin::signed(*ALICE), ORG_NAME1.to_vec()));
+		assert_ok!(Dao::create_organization(Origin::signed(*ALICE), ORG_NAME2.to_vec()));
 
-		// Ensure user 4 is member of 0 organizations
-		assert_eq!(Dao::member_of(4).len(), 0);
+		// Ensure EVE is member of 0 organizations
+		assert_eq!(Dao::member_of(*EVE).len(), 0);
 
-		// Ensure user 4 can be added to a DAO
-		assert_ok!(Dao::add_members(Origin::signed(1), ORG_NAME1.to_vec(), 4));
-		assert_ok!(Dao::add_members(Origin::signed(1), ORG_NAME2.to_vec(), 4));
+		// Ensure EVE can be added to a DAO
+		assert_ok!(Dao::add_members(Origin::signed(*ALICE), ORG_NAME1.to_vec(), *EVE));
+		assert_ok!(Dao::add_members(Origin::signed(*ALICE), ORG_NAME2.to_vec(), *EVE));
 
-		// Ensure the user 4 is member of 2 organizations
-		assert_eq!(Dao::member_of(4).len(), 2);
+		// Ensure the EVE is member of 2 organizations
```

Comment:
I updated the comments to reflect the code changes.

Reply:
Thanks!